Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,186 @@
+
+∆N1 = 5 rpm 
+
+∆N2 = 3 rpm 
+
+∆N2 = 1 rpm 
+
+t1 = 0.25 sec 
+
+t2 = 0.5 sec 
+
+t2 = 1 sec  
+
+Our current goal is for the rotary device to take roughly 3 seconds to transition from 1 compartment to another. The device would need to rotate 18° to have a complete transition. Luckily for us our choice of stepper motor makes it so each step is 1.8°, which makes it so we have to take exactly 10 steps to reach the next compartment. Using this information, we can determine that are desired rpm will be roughly 1 rpm since 18°/3 sec =6° per second.  
+
+To find the motor that would work best for our project  
+
+Stepper Motors are DC motors that divided a full rotation into a set of equal steps. This will give us greater control on speed and movement of our motor, and the math happens to work in our favor.  
+
+To know what type of stepper motor we need, we have to calculate the torque it will need to properly rotate our device.  
+
+With the known values above, and the values that can be varied, a torque calculation table can be created for each case:  
+
+**Givens:**  
+W = 75 lbs  
+R1 = 1.5 ft  
+R2 = 0.67 ft  
+
+**Required Torque Calculations (lb-ft)**  
+
+|  t(s)  |        |∆N (rpm)|        |  
+|--------|--------|--------|--------|
+|        |    1   |    2   |    3   |
+|  0.25  | 1.3122 | 3.9367 | 6.5611 |
+|  0.5   | 0.6561 | 1.9683 | 3.2806 |
+|    1   | 0.3281 | 0.9842 | 1.6403 |  
+
+**Converted to Nm**  
+
+|  t(s)  |        |∆N (rpm)|        |  
+|--------|--------|--------|--------|
+|        |    1   |    2   |    3   |
+|  0.25  | 1.7791 | 5.3374 | 8.8957 |
+|  0.5   | 0.8896 | 2.6687 | 4.4479 |
+|    1   | 0.4448 | 1.3344 | 2.2239 |  
+
+As shown above, the highest amount of torque required would be at 5 rpm, accelerating in 0.25 seconds. This torque would be 8.8957 Nm.  
+However, these results are assuming that a DC motor will be 100% efficient when operating at a lower rated load. This is an unreasonable expectation. Most DC motors are designed to be anywhere from 50-100% efficient with peak efficiency being at around 75% of the rated load. So for a 12 Nm motor, the optimal load range is from 6 to 12 Nm, with peak being at 9 Nm. Because we would have at most 8.8957 Nm of required torque, and an assumed efficiency of 50% the most torque requited would be 17.7914 Nm. However we are deciding to run the motor assuming a lower acceleration time and rpm. At the values we are expecting to run (0.5s acceleration and 3 rpm), also assuming 50% efficiency, the required torque would be 5.3374 Nm, which is an acceptable level for our desired motor. 
+
+^ Source: https://www.energy.gov/sites/prod/files/2014/04/f15/10097517.pdf 
+
+Since the motor we are looking at can have a holding torque of 12 Nm, it will meet the constraints of outputting enough torque and securing the devices. We would be able to operate the motor at a lower demand than its maximum, putting less strain on the motor over time.  
```

Comment:
You did not consider slowing it down. How does this motor slow down the thing once it is rotating? Further, is a break needed for safety?